data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -56,6 +56,7 @@ def test_transform(self, mock_etl):
         # setup - read in sample output as dataframe
```

Comment:
I can't highlight the lines above, but could we get rid of everything that references blocks? E.g., `acs_dst = etl.BLOCK_GROUP_CSV`. 

As with the other comment, this is my fault! I'm the one who didn't update these tests enough when switching to tracts.

Reply:
That's fine - as always - the main work is done but the unit tests hold us up 😮‍💨